src/main/java/com/prgms/springbootbasic/member/model/BlackList.java

```diff
@@ -0,0 +1,43 @@
+package com.prgms.springbootbasic.member.model;
+
+import com.prgms.springbootbasic.global.exception.CantReadFileException;
+import com.prgms.springbootbasic.global.util.ExceptionMessage;
+import com.prgms.springbootbasic.global.util.Application;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Repository;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public class BlackList {
+
+    private static final Logger logger = LoggerFactory.getLogger(BlackList.class);
+    @Value("${file.blacklist}")
+    private String FILE_PATH;
```

Comment:
```suggestion
    @Value("${file.blacklist}")
    private String filePath;
```
상수가 아니라서 카멜케이스 적용하시는게 컨벤션입니다!